src/components/family-search/family-search-results-table/FamilySearchResultsTable.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import {
+  Box,
+  Paper,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+  TableFooter,
+  TableContainer,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import { FamilySearchResponse } from "../../../api/FamilyAPI";
+import { DefaultFields } from "../../../constants/DefaultFields";
```

Comment:
Maybe let's make a list in DefaultFields for the search
```suggestion
import { DefaultFamilySearchFields } from "../../../constants/DefaultFields";
```
inside `DefaultFields` you'll need to make a list like
```
export const DefaultFamilySearchFields: DefaultField[] = [
  DefaultFields.FIRST_NAME,
  DefaultFields.LAST_NAME,
  DefaultFields.PHONE_NUMBER,
  DefaultFields.EMAIL,
  DefaultFields.NUM_CHILDREN,
];
```